frontend/js/helpers/Transaction.js

```diff
@@ -0,0 +1,131 @@
+//common functions that can be used in different cases
+
+
+async function Get(path) {
```

Comment:
I imagine these codes are from Common.js.
If so, please delete these codes and call the same function from Common.js which have already added in development branch.